web/packages/teleterm/src/ui/services/connectMyComputer/connectMyComputerService.ts

```diff
@@ -44,13 +45,16 @@ export class ConnectMyComputerService {
   createRole(
     rootClusterUri: uri.RootClusterUri
   ): Promise<CreateConnectMyComputerRoleResponse> {
-    return this.tshClient.createConnectMyComputerRole(rootClusterUri);
+    return this.terminalServiceClient.createConnectMyComputerRole({
+      rootClusterUri,
+    }).response;
   }
 
   async createAgentConfigFile(rootCluster: Cluster): Promise<void> {
-    const { token } = await this.tshClient.createConnectMyComputerNodeToken(
-      rootCluster.uri
-    );
+    const { token } =
+      await this.terminalServiceClient.createConnectMyComputerNodeToken({
+        rootClusterUri: rootCluster.uri,
+      }).response;
```

Comment:
Responses shouldn't be awaited like this. Look for 

https://github.com/timostamm/protobuf-ts/blob/main/MANUAL.md#generic-rpc-clients

> So a simple way to get the response message of a unary call would be:
> 
> ```
> let call = service.myMethod(foo);
> let response = await call.response;
> ```
> 
> But there is a caveat: gRPC and gRPC web use response trailers to indicate server status. This means that it is possible that the server responds with a message and then sends an error status. If you do not check the status, you may be missing an error.
> 
> Response trailers are a very useful feature, but for simple unary calls, awaiting two promises is cumbersome. For this reason, the UnaryCall itself is awaitable, and will reject if an error status is received. Instead of awaiting call.response, you can simply await the call:
> 
> ```
> let {response} = await service.myMethod(foo);
> ```

Reply:
Oh, right. I will change it.